src/OICodeGen.cpp

```diff
@@ -1247,6 +1353,41 @@ bool OICodeGen::isEmptyClassOrFunctionType(drgn_type *type,
           drgn_type_num_members(type) == 0);
 }
 
+/*
+ * Returns true iff the provided type
+ */
+bool OICodeGen::isDynamic(drgn_type *type) const {
+  if (!config.polymorphicInheritance) {
+    return false;
+  }
+
+  if (!drgn_type_has_virtuality(type)) {
+    return false;
+  }
+
+  if (drgn_type_virtuality(type) != 0 /*DW_VIRTUALITY_none*/) {
+    // Virtual class - not fully supported by OI yet
+    return true;
+  }
+
+  drgn_type_member_function *functions = drgn_type_functions(type);
+  for (size_t i = 0; i < drgn_type_num_functions(type); i++) {
+    struct drgn_qualified_type t {};
```

Comment:
Just a style thing but I think we tend to use no space before the uniform initializer braces (i.e., t{}). Personally I dislike extra spaces.

Reply:
I tried changing this but the linter doesn't like it without a space

Reply:
Ah, it's because I wrote `struct drgn_qualified_type` instead of just `drgn_qualified_type`!